packages/web-react/src/components/Tooltip/__tests__/useTooltipStyleProps.test.ts

```diff
@@ -6,13 +6,13 @@ describe('useTooltipStyleProps', () => {
     const { result } = renderHook(() => useTooltipStyleProps({}));
 
     expect(result.current.classProps.rootClassName).toBe('Tooltip');
-    expect(result.current.classProps.arrowClassName).toBe('Tooltip__arrow');
-    expect(result.current.classProps.closeButtonClassName).toBe('Tooltip__close');
+    expect(result.current.classProps.arrowClassName).toBe('TooltipPopover__arrow');
```

Comment:
Nice-to-have: refactor this test according to the new approach. :-)

Reply:
refactored 👍